deployment/README.md

```diff
@@ -0,0 +1,157 @@
+# Deployment
+
+==============================================
+
+# WARNING
+
+There MAY NOT be any files in this directory that are not committed to GitHub, like a .env file,
+or the asset hash used in the pipeline description will be different than the one that Build stage calculates,
+and Assets stage will not be able to find the assets generated by the Build stage, and thereby fail.
+
+==============================================
+
```

Comment:
Should there be some instruction about creating the .env file? I had to do this to avoid getting an error:

```
Unable to fetch parameters [/production-mmt/CMR_ROOT,/production-mmt/CUMULUS_REST_API,/production-mmt/EARTHDATA_USERNAME,/production-mmt/MMT_ROOT] from parameter store for this account.
```

Reply:
The `production` parameters haven't been populated in the Parameter Store, only the `dit` ones. Maybe you need to set `export MMT_STACK_STAGE="dit"` ?

Reply:
Yes I think I was missing that instruction, somethings got a bit mixed up because I started the instructions on the wrong branch 🤦🏽‍♀️